usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -254,26 +238,40 @@ def handle_column(col, bronze_table_name):
                 regexp_YYYYmmdd = (
                     r"(\\d{4})(?<sep>[-/]?)(\\d{2})(\\k<sep>)(\\d{2})(.\\d{2}:\\d{2}:\\d{2}([+-]\\d{2}:\\d{2})?)?"
                 )
```

Comment:
I'd say move these regex strings outside the case statements, so `parse_string_date` and `truncate_string_date` can be top level cases. This will prevent the need for nested `if` statements